src/config.rs

```diff
@@ -2227,14 +2226,16 @@ pub struct CdcConfig {
 
 impl Default for CdcConfig {
     fn default() -> Self {
+        let total_mem = SysQuota::memory_limit_in_bytes();
         Self {
             min_ts_interval: ReadableDuration::secs(1),
             hibernate_regions_compatible: true,
             scan_lock_pool_size: 2,
             // TiCDC requires a SSD, the typical write speed of SSD
             // is more than 500MB/s, so 128MB/s is enough.
             incremental_scan_speed_limit: ReadableSize::mb(128),
-            old_value_cache_capacity: OptionReadableSize(None),
+            // 10% total memory by default.
```

Comment:
Could you set an upper limit of cache quota? In most cases, I think 512MB is enough.